src/pages/admin/DynamicForm/FormCreate.js

```diff
@@ -158,6 +158,8 @@ const FormCreateForm = (props) => {
   } = useParams();
   const [viewUserForm, setViewUserForm] = useState(true);
 
+  const isEditMode = !!eventId; // Check if in edit mode or not
```

Comment:
super small detail, lets change this condition so that it checks for both eventId AND eventYear, so that it aligns with what our primary key is in the database.

Reply:
actually I just noticed that we have a constant called isSaved, which is exactly !!(eventID && eventYear), so lets remove isEditMode and just use isSaved instaed

Reply:
agree with this @stevenh17

Reply:
oops mb didnt see I just made the change and pushed